src/main/java/com/mongodb/jdbc/MongoJsonSchema.java

```diff
@@ -48,16 +66,18 @@ public static MongoJsonSchema createScalarSchema(String type) {
      * @return void
      */
     @SafeVarargs
-    public final void addRequiredScalarKeys(Pair<String, String>... scalarProperties) {
+    public final void addRequiredScalarKeys(ScalarProperties... scalarProperties) {
```

Comment:
The comments describing this method need an update with the parameter change.

Reply:
That's very true, the javadoc and method name are totally out-of-date :D
Done.